### ISSUE TITLE
홈페이지 리뉴얼 필요

### ISSUE THREAD

**Issue by ehgus** [juliakorea/juliakorea.github.io#13]
현재 한국 julia 홈페이지는 이전에 사용하던 포맷을 따르고 있습니다. 
https://julialang.org/ 의 레이아웃이 크게 바뀌었기 때문에 이에 따라 한국 홈페이지도 변화해야 할 필요가 있다고 생각합니다.

https://cn.julialang.org/ 을 롤모델 삼아 바꾸면 될 것 같습니다.

**Comment by wookay** [comment]
네. https://github.com/juliakorea/translate-home  요기에 소스는 있습니다. renewal branch 새로 만들어도 좋을것 같네요.  페이스북 그룹에 오셔서 최근 번역 작업이랑 계획 같은것도 소개해 주시면 참 좋을 것 같습니다.
